test/metabase/upload_test.clj

```diff
@@ -329,9 +344,10 @@
 (defn do-with-upload-table! [table thunk]
   (try (thunk table)
        (finally
-         (driver/drop-table! driver/*driver*
-                             (:db_id table)
-                             (#'upload/table-identifier table)))))
+         (when (not= driver/*driver* :redshift) ; redshift tests flake when tables are dropped
```

Comment:
Are they eventually cleaned out somehow?

Reply:
yeah, redshift session schemas are dropped at the end of the test run. So as long as we use the session schema to create the tables, these will be cleaned up. Might be worth a comment here but this is something everyone should know if they've developing redshift/snowflake/oracle/bigquery or any of the other cloud DBs